projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-get-media-v1-1-endpoint.php

```diff
@@ -1,47 +1,51 @@
 <?php
 
-new WPCOM_JSON_API_Get_Media_v1_1_Endpoint( array(
```

Comment:
I only fixed the linting/formatting issues here, that's why we see a large diff generated.